<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: url(:/res/background/background.png)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCash">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Cash:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="bet">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>540</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerFirstCard">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>93</width>
      <height>132</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pScore">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>350</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgba(28,28,28,0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Score_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dScore">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>210</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgba(28,28,28,0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCard">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>360</y>
      <width>132</width>
      <height>132</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgba(28,28,28,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="playerCash">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgba(28,28,28,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBet">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Bet:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stand">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>132</width>
      <height>132</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgba(28,28,28,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="dealerFirstCard">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>93</width>
      <height>132</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startGame">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>275</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgba(28,28,28,0);</string>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="Score_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deckCard">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>93</width>
      <height>132</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/res/background/backgroundCard.png);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
